cjworkbench/i18n/trans.py

```diff
@@ -169,24 +138,24 @@ def _process_simple_message(self, message, fallback, parameters={}):
         if message:
             try:
                 return self._format_message(
-                    message, parameters=parameters, do_escape=False
+                    message, parameters=parameters, html_escape=False
                 )
-            except (ICUError, InvalidICUParameters):
+            except Exception:
```

Comment:
I much preferred the `(ICUError, InvalidICUParameters)`. Why did you switch back?

Reply:
I did prefer it too, but I realized that the only way I was creating `InvalidICUParameters` was a
```
except Exception:
   raise InvalidICUParameters
```
in the function that passes the parameters to ICU, without having any idea what `Exception` might be.

Reply:
Oh, awesome! You nixed the exception altogether :).

The `except: pass` thing threw me for a loop. Here's what I think would be more natural:

At top of file:

```
import logging
...
logger = logging.getLogger(__name__)
```

and here:

```
try:
    return self._format_message(...)
except ICUError:
    logger.exception("Error in .po file")
    # ... and fall through to default
return self._format_message(...)
```

This strikes me as safer, because I'll be emailed at least once when there's an error :). "Crash + email Adam" is far, far better than "Never crash and never expose errors."

Reply:
(I don't like to catch `Exception`. Let's enumerate the errors.)

Reply:
I added the logging in pull request #164, since it's already (and still) open, in order not to open a new pull request